source/language/delays.rst

```diff
@@ -297,27 +295,27 @@ within a box are implementation details of the box; they are invisible
 to the outside scope and therefore do not prevent commutation.
 
 We introduce a ``boxas`` expression for labeling a box. We primarily use this to
-later refer to the length of this box. Boxed expressions are good for
+later refer to the duration of this box. Boxed expressions are good for
 this because their contents are isolated and cannot be combined with
 gates outside the box. Therefore, no matter how the contents of the box
-get optimized, the ``lengthof(boxlabel`` has a well-defined meaning.
+get optimized, the ``lengthof(boxlabel)`` has a well-defined meaning.
 
 .. code-block:: c
 
        boxas mybox {
            cx q[0], q[1];
            delay[200ns] q[0];
        }
-       delay[length(mybox)] q[2], q[3];
+       delay[duration(mybox)] q[2], q[3];
        cx q[2], q[3];
 
 We introduce a ``boxto`` expression. The contents of it will be boxed, and in
 addition a total duration will be assigned to the box. This is useful
 for conditionals where the box will declare a hard deadline. The natural
-length of the box must be smaller than the declared boxto duration,
+duration of the box must be smaller than the declared boxto duration,
 otherwise a compile-time error will be raised. The stretch inside the
 box will always be set to fill the difference between the declared
-length and the natural length.
+duration and the natural duration.
```

Comment:
@levbishop says that `boxto` should be dropped, as it is equivalent to `box[<duration>]`.

Reply:
Addressed in https://github.com/Qiskit/openqasm/pull/204/commits/f56961122910730ee52c6b091dc1cfedfc28986b.